js/foundation/foundation.tooltip.js

```diff
@@ -234,8 +234,6 @@
         }).end();
       };
 
-      objPos(tip, (target.offset().top + target.outerHeight() + 10), 'auto', 'auto', target.offset().left);
```

Comment:
This line was redundant and could generate some problems in the calculation of "left" value